component/table.jsx

```diff
@@ -0,0 +1,48 @@
+import { css } from '../lib/dom.js'
+
+css(`
+.container {
+  overflow-x:auto;
+}
+.customers {
```

Comment:
`.customers` is not good either since this component is not specific to the customers table but a generic one, you can rely on a generic tag + a prefix like `table.nan`

Reply:
also, you called the page students and here it's customers, we need to avoid mixing names as much as possible